src/classes/person.rb

```diff
@@ -5,7 +5,7 @@ class Person < Nameable
   attr_accessor :name, :age, :rentals
   attr_reader :id, :type
 
-  def initialize(age, name = 'Unknown', parent_permission: true)
+  def initialize(age, name, parent_permission: true)
```

Comment:
- [ ] Good work displaying the options, but when I try to create a person, an error occurs. Kindly check it out.

![image](https://user-images.githubusercontent.com/98453979/224475152-55657fdf-e18f-4212-a76e-9baba805ea07.png)

![image](https://user-images.githubusercontent.com/98453979/224475181-f8fc45bf-0d48-40af-a7bd-7e5cf5ca1f94.png)